manager/src/test/java/org/apache/hertzbeat/manager/service/MonitorServiceTest.java

```diff
@@ -652,9 +653,8 @@ void getMonitorDto() {
 
     @Test
     void getMonitors() {
-        Specification<Monitor> specification = mock(Specification.class);
-        when(monitorDao.findAll(specification, PageRequest.of(1, 1))).thenReturn(Page.empty());
-        assertNotNull(monitorService.getMonitors(specification, PageRequest.of(1, 1)));
+        doReturn(Page.empty()).when(monitorDao).findAll(any(Specification.class), any(PageRequest.class));
+        assertNotNull(monitorService.getMonitors(null, null, null, null, null, null, null, 1, 1, null));
```

Comment:
Seems this test case is not match param list of `getMonitors`. Some params have a default value.